types/audit.d.ts

```diff
@@ -70,17 +70,43 @@ declare global {
       wastedPercent?: number;
     }
 
+    export interface ClientRectOverlapFailure  {
+      overlapScoreRatio: number;
+      tapTargetScore: number;
+      overlappingTargetScore: number;
+    }
+    
+    export interface TapTargetOverlapFailure extends ClientRectOverlapFailure {
+      tapTarget: Artifacts.TapTarget;
+      overlappingTarget: Artifacts.TapTarget;
+    }
```

Comment:
are these used only in the audit? If so, should move them as typedefs in that file as they aren't really needed in the audit type definition file (I think you still can't use `extends` for typedefs in js, unfortunately, but it's just three properties so not a big deal to repeat).

`ClientRectOverlapFailure` may not even need to be a typedef if it looks ok inline in that `@return` statement

Reply:
Makes sense, they don't need to be in there 👍